conf/app.ini

```diff
@@ -23,6 +23,8 @@ PULL_REQUEST_QUEUE_LENGTH = 1000
 PREFERRED_LICENSES = Apache License 2.0,MIT License
 ; Disable ability to interact with repositories by HTTP protocol
 DISABLE_HTTP_GIT = false
+; Force ssh:// clone url instead of scp-style uri when default SSH port is used
+FORCE_CLONE_SSH_URL = false
```

Comment:
`FORCE_CLONE_SSH_URL` might be a little bit misleading because it could also mean that the entered url is the forced clone ssh url. I would use `USE_SSH_URI_SCHEME`.